bokehjs/src/coffee/models/mappers/log_mapper.coffee

```diff
@@ -20,11 +21,14 @@ export class LogMapper extends Model
     if inter_scale == 0
       value = 0
     else
-      value = (Math.log(x) - inter_offset) / inter_scale
-      if isNaN(value) or not isFinite(value)
-        value = 0
+      clamped_x = clamp(x, @source_range.start, @source_range.end)
```

Comment:
I'm not clamping the opposite path (`map_from_target` and `v_map_from_target`). I'm not sure if it's necessary to do that.